Assets/Scenes/Scripts/SetHealthBar.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+//attach to Healthbar
+public class SetHealthBar : MonoBehaviour 
+{
+	[SerializeField]
+	private Sprite HealthBar100;
```

Comment:
This is hard coding to the extreme. If I wanted it to be be able to show health at 70% you would have to add another variable. If I then wanted to show health at 75%, you would have to add another variable. It gets out of control to the point where you would have 100 variables for every possible number. Instead, just make this configurable and display a number of the current health or something if you can't come up with a way to do a proper health bar.